macros/drop_schemas_automation.sql

```diff
@@ -0,0 +1,30 @@
+{% macro drop_schemas_automation(drop_target_schema=true) %}
+    {{ return(adapter.dispatch('drop_schemas_automation')(drop_target_schema)) }}
+{%- endmacro %}
```

Comment:
Out of curiosity, is there a reason this dispatch looks different from the one used in the quote macro? Should we be including the fivetran_utils dispatch in here?

I have always been fuzzy with my understanding of the dispatch protocol. However, I think we should structure both to be in line with the latest structure of the dbt-utils macros. [Here](https://github.com/dbt-labs/dbt-utils/blob/9507d6bcd6c04b73e174a9aaee11865f9387e096/macros/sql/star.sql#L1-L3) is what they have for the `star` macro.

I believe adding `fivetran_utils` will ensure that the macro will only be used when prefixed with fivetran_utils and could not be at risk of overriding a users own macro. This is what I got from their [documentation](https://docs.getdbt.com/reference/dbt-jinja-functions/dispatch#for-package-maintainers), interested to get your input on this as well.

Reply:
added